freshmaker/logger.py

```diff
@@ -102,22 +120,22 @@ def setup_logger():
         backupCount=1,
         mode="a",
     )
-    file_handler.setLevel(LOG_LEVEL)
+    file_handler.setLevel(conf.log_level)
     file_handler.setFormatter(file_formatter)
     logger.addHandler(file_handler)
     if os.environ.get("CONSOLE_LOGS", "").lower() != "false":
         console_handler = logging.StreamHandler(stream=sys.stdout)
-        console_handler.setLevel(LOG_LEVEL)
+        console_handler.setLevel(conf.log_level)
         console_handler.setFormatter(
             logging.Formatter(
                 "[%(asctime)s.%(msecs)d] [%(processName)s - %(threadName)s] "
                 "[%(levelname)s] %(message)s",
                 "%Y-%m-%dT%H:%M:%S",
             )
         )
+        remove_root_default_handler()  # avoid duplicated log lines due to two handlers
```

Comment:
Good point! I moved the console handler to the root logger:

https://github.com/redhat-exd-rebuilds/freshmaker/blob/b3096339a89c1a607230bd7fbe4950db02a6e073/freshmaker/logger.py#L126-L161

I left the console handler part out of the if clauses. I tried to keep the same functionality as we had before, but it does look a bit weird to me. In case log_backend=journal, in the older version we would still have the stream handler  in the root logger because of a `basicConfig` call, so I kept that for now. Do you think that is okay? Or should I move it to the first if block?

Also, I made the last case more explicit and required `log_backend=file`, and raised an exception for unrecognized clauses. In the older version, if for example someone mistyped `log=console="cosnole"`, they would end up with a file backend and no error, which i think is a bit weird...

Reply:
> I left the console handler part out of the if clauses. I tried to keep the same functionality as we had before, but it does look a bit weird to me. In case log_backend=journal, in the older version we would still have the stream handler in the root logger because of a basicConfig call, so I kept that for now. Do you think that is okay? Or should I move it to the first if block?

ok, I missed this comment when approve it and posted my comment, I agree it looks a little weird and +1 to move it to the "console" branch. We also need it in "journal" branch, because freshmaker is currently deployed with log backend "journal", and we do want to have logs in console, a more ideal solution would be changing the deployment to use "console" log backend and remove the "journal" support, I have no idea why it was there and use it as default in our deployment.

Reply:
I removed the 'journal' and 'file' log backends and their related configurations, that will make the code simpler! I also updated the commit message to reflect that.